src/webui/webapplication.cpp

```diff
@@ -683,6 +683,8 @@ void WebApplication::sessionStart()
     QByteArray cookieRawForm = cookie.toRawForm();
     if (m_isCSRFProtectionEnabled)
         cookieRawForm.append("; SameSite=Strict");
+    else
+        cookieRawForm.append("; SameSite=None");
```

Comment:
Probably an error will be raised, according to https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie:
>None
>
>    means that the browser sends the cookie with both cross-site and same-site requests. The Secure attribute must also be set when setting this value, like so SameSite=None; Secure. If Secure is missing an error will be logged: 

So you should set the attribute to `None` only when `cookie.isSecure()` is `true`.

Reply:
ok, here is new commit